pkg/controller/machineset/machineset_controller.go

```diff
@@ -59,6 +59,10 @@ const (
 	// Ansible execution.
 	jobPrefix = "provision-machineset-"
 
+	// deprovisionJobPrefixPrefix applies a prefix to the standard job prefix when we're
+	// running a deprovision playbook. (sorry)
+	deprovisionJobPrefixPrefix = "de"
```

Comment:
The JobControl is created with a specific job prefix which it looks for. If jobs are created with a different prefix, then the JobControl will not be able to manage them.

Reply:
Seemed to be working in practice and all tests but must be something not covered in testing.